### ISSUE TITLE
False-positive of WRONG_NEWLINES - functional style on fully qualified names

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#321]
```kotlin
java.lang.Boolean.getBoolean(properties.getProperty("parallel.mode"))
```
FQ name `java.lang.Boolean` is detected as a chain of calls by the rule.
